app/enterprise/1.5.x/kong-for-kubernetes/deployment-options.md

```diff
@@ -147,7 +147,7 @@ controller can re-create Kong's proxy configuration even if the underlying
 datastore changes.
 
 While most Kubernetes resources can be left unchanged when migrating between
-deployment types, users must remove any KongPlugin resources that use
+deployment types, users must remove any Kong Plugin resources that use
```

Comment:
```suggestion
deployment types, users must remove any KongPlugin resources that use
```
This should remain as-is. `KongPlugin` is the resource name: https://github.com/Kong/kubernetes-ingress-controller/blob/master/docs/guides/using-kongplugin-resource.md